source/specifications.rst

```diff
@@ -9,11 +9,58 @@ toolchain, the Python Packaging Authority is also responsible for maintaining
 the interoperability specifications used to define the interactions between
 those tools.
 
-Specification Process
+Active Specifications
 ---------------------
 
-PyPA interoperability specifications are currently handled as Informational
-Python Enhancement Proposals in accordance with :pep:`1`.
+The currently active specifications are recorded in the
+:ref:`PyPA Specifications <pypug:specifications>` section of the
+Python Packaging User Guide.
+
+
+Specification Update Process
+----------------------------
+
+PyPA interoperability specifications are separated into two categories:
+
+* `Package Distribution Metadata <https://packaging.python.org/specifications/#package-distribution-metadata>`_
+* `Package Index Interfaces <https://packaging.python.org/specifications/#package-index-interfaces>`_
+
+For Package Distribution Metadata, the default responsible decision maker is
+the lead CPython representative on distutils-sig.
+
+For Package Index Interfaces, the default responsible decision maker is
+the lead maintainer for the `Python Package Index <https://pypi.org>`__.
+
+The current lead PyPI maintainer is Donald Stufft, while the current lead
+CPython representative on distutils-sig is Nick Coghlan.
+
+
+Handling fixes and other minor updates
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Backwards compatible fixes and small enhancements to the specifications are
+handled by submitting pull requests to the specifications section of the guide.
```

Comment:
I'm not sure this is actually true for the *new* PEPs is it? We amended PEP 440 and PEP 503 directly. AFAIK this is really only true for minor changes to the old status quo where we either don't have a PEP, or if we do we already diverge from it in some way.